src/__tests__/update-logs.test.ts

```
@@ -0,0 +1,352 @@
+
+			expect(logEntry).toMatchObject({
+				level: 50, // Error level
+				msg: expect.stringContaining('Error message'),
+				context: 'error_context',
+				source: 'ratos-update'
+			});
+		} finally {
+			if (originalLogFile) {
+				process.env.RATOS_LOG_FILE = originalLogFile;
+			} else {
+				delete process.env.RATOS_LOG_FILE;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)